scripts/simulator_base.js

```diff
@@ -2551,6 +2569,10 @@ var SIMULATOR = {};
 					continue;
 				}
 
+				if (current_assault.bash) {
```

Comment:
I like the idea of extracting passive skills into separate functions.  We should probably create a `passive_skills` object to hold them though, rather than add non-activated skills to `activated_skills`.